nixos/modules/profiles/installation-device.nix

```diff
@@ -51,22 +51,23 @@ with lib;
     services.mingetty.helpLine = ''
       The "nixos" and "root" accounts have empty passwords.
 
-      Type `sudo systemctl start sshd` to start the SSH daemon.
-      You then must set a password for either "root" or "nixos"
-      with `passwd` to be able to login.
+      An ssh daemon is running. You then must set a password
+      for either "root" or "nixos" with `passwd` or add an ssh key
+      to /root/.ssh/authorized_keys be able to login.
```

Comment:
Shouldn't we recommend they add the key to the *nixos* user instead?

The thought comes from the fact we're running the installer medium using the *nixos* user, and not *root* anymore.

Reply:
I can change it.

Reply:
Fixed.